chainer/links/connection/lstm.py

```diff
@@ -70,6 +79,11 @@ def __call__(self, c, h, x):
                 output of LSTM units.
 
         """
+        if self.upward.has_uninitialized_params:
+            in_size = x.size // len(x.data)
```

Comment:
Is this line needed? I am not sure if the following two lines really need `in_size`. For example, `self._initialize_params(in_size)` does not seem to use `in_size` anywhere.